docs/tutorial/quantum_annealing.ipynb

```diff
@@ -0,0 +1,446 @@
+   "execution_count": null,
+   "id": "bf8c2ace-0bf3-4a87-9d27-fe7bde7a8bac",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "T = 100.0\n",
+    "def f0(t, args):\n",
+    "    return t/T\n",
+    "\n",
+    "def f1(t, args):\n",
+    "    return 1 - t/T"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1f1084c-02de-4d96-ba4b-b23cc9ef798e",
+   "metadata": {},
+   "source": [
+    "The entire Hamiltonian is given as a list where each element is a pair consisting of a Hamiltonian and its coefficient.\n",
+    "\n",
+    "We create the initial state $ \\psi(t)$ at $t = 0$, which is the ground state of $H_1$.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a61ce5bd-9064-4b2c-9890-a4251cd18f0f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "H = [[qt_H0, f0], [qt_H1, f1]]\n",
+    "psi0 = tensor([Qobj([[1], [1]])/np.sqrt(2) for i in range(V)])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eeaac1de-4c86-4d6a-a3d7-a9f2dc983cca",
+   "metadata": {},
+   "source": [
+    "This time, we will not only look at the success probability but also examine how much $|\\psi(t)\\rangle$ overlaps with $|\\phi_0\\rangle$ at each moment. To solve the Schrödinger equation, we use `mesolve`. By providing the Hamiltonian to be solved (which may depend on time), the initial state, a list of times, a list of collapse operators (which is not relevant this time), and a list of operators for which we want to calculate the expectation values at each moment (this time, $\\rho = |\\phi_0\\rangle \\langle \\phi_0|$, a function that converts the ket vector into a density operator), the dynamics can be calculated. `ket2dm` converts the ket vector to a density operator.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "06cccc3f-42c8-432a-9aae-67ca4daa5686",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tlist = np.linspace(0.0, T, 101)\n",
+    "result = mesolve(H, psi0, tlist, [], [ket2dm(eket0)])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9e4aa0f7-2a04-4161-8563-79aad9008883",
+   "metadata": {},
+   "source": [
+    "Among the calculation results, the expected value $|\\langle \\phi_0 | \\psi(t) \\rangle|^2$, which we wanted to calculate, is stored in `result.expect[0]`. Let's plot it.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "80665cbf-18d0-4f2b-a0da-e038c108d5dd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.plot(result.times, result.expect[0])\n",
+    "plt.xlabel(\"Time\")\n",
+    "plt.ylabel(\"Probability\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e646589d-cd63-4847-adb5-2c74fc89e36b",
+   "metadata": {},
+   "source": [
+    "Masaphys. *PythonとQuTiPで量子アニーリングのダイナミクスを追う*. Qiita. Accessed October 30, 2024. [https://qiita.com/masaphys/items/38aeaee5313f81e15414](https://qiita.com/masaphys/items/38aeaee5313f81e15414)\n"
```

Comment:
Refer the blog is OK, but Please do not copy someone's blog post.
So please rewrite the contents and codes